Tests/CarthageKitTests/XcodeSpec.swift

```diff
@@ -394,28 +379,6 @@ class XcodeSpec: QuickSpec {
 			let result = ProjectLocator.locate(in: directoryURL.appendingPathComponent("ReactiveCocoaLayout")).first()
 			expect(result).to(beNil())
 		}
-
-		it("should symlink the build directory") {
-			let dependency = Dependency.gitHub(.dotCom, Repository(owner: "github", name: "Archimedes"))
-			let version = PinnedVersion("0.1")
-
-			let dependencyURL =	directoryURL.appendingPathComponent(dependency.relativePath)
-			// Build
-			let buildURL = directoryURL.appendingPathComponent(Constants.binariesFolderPath)
-			let dependencyBuildURL = dependencyURL.appendingPathComponent(Constants.binariesFolderPath)
-
-			let result = build(dependency: dependency, version: version, directoryURL, withOptions: BuildOptions(configuration: "Debug"))
-				.flatten(.concat)
-				.ignoreTaskData()
-				.on(value: { project, scheme in // swiftlint:disable:this end_closure
-					NSLog("Building scheme \"\(scheme)\" in \(project)")
-				})
-				.wait()
-
-			expect(result.error).to(beNil())
-
-			expect(dependencyBuildURL).to(beRelativeSymlinkToDirectory(buildURL))
-		}
```

Comment:
I deleted this test because symlinks are no longer handled by the `build` method that's being tested here.